### ISSUE TITLE
No empty file check for tsx files

### ISSUE THREAD

**Issue by Felix-kln** [sindresorhus/eslint-plugin-unicorn#2001]
Hello, I don't know if a bug report is the right thing here but I wanted to use the no-empty-file rule for my react project.
However is does not check .tsx files.
Is it possible to add these to the files to check?

Rule: no-empty-file

empty files:

test.ts // es lint error (no empty file)
tester.tsx // no error

**Comment by fisker** [comment]
`.tsx` and `.jsx` is not added, because I'm not sure what's the result of

```js
import foo from './empty.jsx';
```

OR

```js
import * as foo from './empty.jsx';
```

, and not sure if it reasonable to use.

**Comment by Felix-kln** [comment]
I played around a bit in the node module and when I added the tsx in the `if(!/\.(?:js|....|tsx)$/.test(filename)` my eslint gave me the error aswell for my tsx file.

**Comment by fisker** [comment]
I guess we can add them to the pattern. Would you like to work on it?

**Comment by fisker** [comment]
Good, can you also add `i` to the regexp, I think we should also check `FOO.JS`.
